pkg/ceph/osd/daemon.go

```diff
@@ -38,6 +38,11 @@ const (
 )
 
 func Run(context *clusterd.Context, agent *OsdAgent) error {
+	if err := setHostName(context); err != nil {
```

Comment:
ahh. then `NODENAME` is a more appropriate name for the env vs. `HOSTNAME` which is a standard linux env.